diff --git a/ehtim/parloop.py b/ehtim/parloop.py
index 942c687..1ea8330 100644
--- a/ehtim/parloop.py
+++ b/ehtim/parloop.py
@@ -50,8 +50,8 @@ class Parloop(object):
             except KeyboardInterrupt:
                 print('\ngot ^C while pool mapping, terminating')
                 pool.terminate()
-                print 'pool terminated'
-            except Exception, e:
+                print('pool terminated')
+            except Exception as e:
                 print('\ngot exception: %r, terminating' % (e,))
                 pool.terminate()
                 print('pool terminated')
